app/enterprise/2.1.x/deployment/installation/ubuntu.md

```diff
@@ -68,17 +68,16 @@ You should now have two files in your home directory on the target system:
 
 ## Step 2. Install Kong Enterprise
 
-1. Update APT and install dependencies
+1. Update APT
 
     ```bash
     $ sudo apt-get update
-    $ sudo apt-get install openssl libpcre3 procps perl
     ```
 
 2. Install Kong Enterprise
 
     ```bash
-    $ sudo dpkg -i kong-enterprise-edition-<VERSION_NUMBER>.deb
+    $ sudo apt-get install $(realpath kong-enterprise-edition-{{site.data.kong_latest.version}}.*.deb)
```

Comment:
Should the .deb package path stay as `$(realpath)` or do we write something like `/path/to/.deb` ? For context, `apt-get` needs the absolute path to the `.deb` package.

Reply:
I'd go with `/path/to/.deb`, that's the style we use most commonly in the rest of the docs.

Reply:
@lena-larionova like this https://github.com/Kong/docs.konghq.com/pull/2321/commits/83602b697338627ed10bc72b9ae7b7f273d89e25?